dsls/sql/src/main/java/org/apache/beam/dsls/sql/interpreter/operator/math/BeamSqlSignExpression.java

```diff
@@ -37,7 +37,13 @@ public BeamSqlSignExpression(List<BeamSqlExpression> operands) {
     BeamSqlPrimitive result = null;
     switch (op.getOutputType()) {
```

Comment:
let's specify the outputType for `BeamSqlSignExpression` in the constructor: 

```java
  public BeamSqlSignExpression(List<BeamSqlExpression> operands) {
    super(operands);
  }
```

to 

```java
  public BeamSqlSignExpression(List<BeamSqlExpression> operands) {
    super(operands, op(0).getOutputType());
  }

```